app/javascript/utils/applicationDetailsUtils.js

```diff
@@ -6,3 +6,8 @@ export const totalSetAsidesForPref = (listing, prefName) =>
   listing.listing_lottery_preferences.find((pref) =>
     pref.lottery_preference.name.includes(prefName)
   )?.available_units ?? 0
+
```

Comment:
should this go in the supp app directory? Might be clearer if renamed to leaseUpApplicationDetailsUntils?